tendrl/monitoring_integration/sync/dashbaord_sync.py

```diff
@@ -47,118 +63,118 @@ def refresh_dashboard(self):
                        {'message': "Failed to update cluster "
                        "dashboard.err: %s" % str(ex)})
 
-    def update_dashboard(self, cluster_details, dashboards):
-        for dashboard_name in dashboards:
-            if dashboard_name in cluster_details and \
-                    cluster_details[dashboard_name]:
-                # check alert dashboard exist
-                alert_dashboard = alert_utils.get_alert_dashboard(
-                    dashboard_name
-                )
-                flag = True
-                if "message" in alert_dashboard and \
-                        alert_dashboard["message"] == "Dashboard not found":
-                    flag = False
-                if flag:
-                    try:
-                        rows_len = len(
-                            alert_dashboard["dashboard"]["rows"]
+    def create_alert_dashboard(self, cluster_details):
+        for dashboard_name in cluster_details:
+            # check alert dashboard exist
+            resource_json = alert_utils.get_alert_dashboard(
+                dashboard_name
+            )
+            dashboard_found = True
+            if "message" in resource_json and \
+                    resource_json["message"] == "Dashboard not found":
```

Comment:
I feel rather use `"Dashboard not found" in resource_json["message"]`

Reply:
i will change it